### ISSUE TITLE
Personachat: Number of persona in training set is more than the number reported in the paper.

### ISSUE THREAD

**Issue by hqsiswiliam** [facebookresearch/ParlAI#3967]
Hi, the number of persona in the training set is 8784, more than the paper's reported version: 1151. Was the dataset updated after the paper was published?

Link for the dataset: [http://parl.ai/downloads/personachat/personachat.tgz](http://parl.ai/downloads/personachat/personachat.tgz)
Link for the paper: [https://arxiv.org/abs/1801.07243](https://arxiv.org/abs/1801.07243)

**Comment by stephenroller** [comment]
No. You're not the first to bring this up, and the paper's count has never been perfectly replicated. But a large percentage of the difference comes from shuffling/dropout/minor text variations of wording. If you attempt to account for most of this, you will be much closer.
